fe/src/main/java/org/apache/doris/common/FeMetaVersion.java

```diff
@@ -179,6 +179,8 @@
     public static final int VERSION_83 = 83;
     // add storage format in schema change job
     public static final int VERSION_84 = 84;
+    // support define expr in materialized view
```

Comment:
The bitmap_union mv should be forbidden now. So the meta version could not be changed.